src/test/java/uk/gov/pay/api/utils/Payloads.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.pay.api.utils;
+
+public class Payloads {
```

Comment:
Perhaps a builder pattern with the defaults from the no args constructor would offer more flexibility, but no need for greater complexity if you don't feel its needed. Just a passing thought.

Reply:
yeah i think that will make things a bit too complex for now